lib/ShareManifests.js

```diff
@@ -0,0 +1,108 @@
+    "name": "Facebook",
+    "origin": "https://www.facebook.com",
+    "pageSize": {
+      "share": {
+        "height": 400,
+        "width": 700
+      }
+    },
+    "shareURL": "https://www.facebook.com/sharer/sharer.php?u=%{url}",
+    "version": 2,
+    "postActivationURL": "https://activations.cdn.mozilla.net/en-US/activated/facebook.html"
+  },
+  "social.manifest.mg-mail.yahoo.com": {
+    "author": "Yahoo! Inc",
+    "description": "Quickly send web pages to yourself and others with enhanced link previews and more.",
+    "homepageURL": "http://mail.yahoo.com",
+    "icon32URL": "https://s.yimg.com/nq/nr/icons/ymail32.png",
+    "icon64URL": "https://s.yimg.com/nq/nr/icons/ymail64.png",
+    "iconURL": "https://s.yimg.com/nq/nr/icons/ymail16.png",
+    "name": "Yahoo Mail",
+    "origin": "https://mg.mail.yahoo.com",
+    "shareURL": "https://mg.mail.yahoo.com/d/app/compose?url=%{url}&title=%{title}",
```

Comment:
no, those are optional.  I only used them for situations where a share page does weird stuff and we cannot size the iframe based on then content.